contracts/scripts/native_solc_compile_all_workflow

```diff
@@ -11,12 +11,12 @@ OPTIMIZE_RUNS=1000000
 
 SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
 
-pip3 install --upgrade pip
-python3 -m pip install --require-hashes -r "$SCRIPTPATH"/requirements.txt
+# pip3 install --upgrade pip
+# python3 -m pip install --require-hashes -r "$SCRIPTPATH"/requirements.txt
 
-if ! solc-select versions | grep -q "$SOLC_VERSION"; then
-  solc-select install $SOLC_VERSION
-fi
+# if ! solc-select versions | grep -q "$SOLC_VERSION"; then
+  # solc-select install $SOLC_VERSION
+# fi
```

Comment:
@MStreet3 I'm guessing this is unintentional :) ?

Reply:
Yea was a hack to fix a compilation issue